### PR TITLE
Update paste to 2.2.1

### PR DESCRIPTION
**Pull request by gobinathm**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/paste.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  sha256 '9e6983c2d6d6e1146a6f2ea5c3cdd7afe309fa7026882be159ef64cd052566df'
+  version '2.2.1'
+  sha256 '131e1a2c06e88c1532570e4342073b043023fd049508b31be9d90a93f6defab1'
 
-  # hockeyapp.net/api/2/apps/ee24d1a939cd4ff8b2861eb8c788a995 was verified as official when first introduced to the cask
-  url 'https://rink.hockeyapp.net/api/2/apps/ee24d1a939cd4ff8b2861eb8c788a995?format=zip'
-  appcast 'https://rink.hockeyapp.net/api/2/apps/ee24d1a939cd4ff8b2861eb8c788a995',
-          checkpoint: '0618aaad6d8f42b92351aedc35feb8873f7b7cd6e38427d13b7cfe56518d8d59'
+  # rink.hockeyapp.net/api was verified as official when first introduced to the cask
+  url "https://rink.hockeyapp.net/api/2/apps/ee24d1a939cd4ff8b2861eb8c788a995/app_versions/6?format=zip"
```
Comment (vitorgalvao) [inline Casks/paste.rb]: The `2` here is the version of `hockeyapp`’s API. It is not related to the app.

Comment (gobinathm) [inline Casks/paste.rb]: Thanks

File: Casks/paste.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
 cask 'paste' do
-  version '2.0.0'
-  sha256 '9e6983c2d6d6e1146a6f2ea5c3cdd7afe309fa7026882be159ef64cd052566df'
+  version '2.2.1'
+  sha256 '131e1a2c06e88c1532570e4342073b043023fd049508b31be9d90a93f6defab1'
 
-  # hockeyapp.net/api/2/apps/ee24d1a939cd4ff8b2861eb8c788a995 was verified as official when first introduced to the cask
```
Comment (victorpopkov) [inline Casks/paste.rb]: I don't think that the vendor comment should be changed as the `ee24d1a939cd4ff8b2861eb8c788a995` part points to the vendor of this app and should be included.

Comment (gobinathm) [inline Casks/paste.rb]: Will role back that

Comment (gobinathm) [inline Casks/paste.rb]: @victorpopkov its been rolled back

**Pull request merged**